tools/src/main/java/org/apache/kafka/tools/PushHttpMetricsReporter.java

```diff
@@ -96,7 +97,31 @@ public class PushHttpMetricsReporter implements MetricsReporter {
                             "producer/consumer/streams/connect instance");
 
     public PushHttpMetricsReporter() {
-        time = Time.SYSTEM;
+        // This is for backward compatibility with older versions like 0.8 and 0.9,
```

Comment:
Could you please includes following items in docs:

1. this is used by xxx E2E
2. the E2E run this report with older client jars
3. hence this class must be compatible with older client jars.

Reply:
> another fix is that we can change time from Time to Supplier<Long> to offer current time. WDYT?

Thanks for the review, this one looks better.